### PR TITLE
Add google-scholar social icon

### PR DESCRIPTION
**Pull request by ShadowMitia**
From https://github.com/xdanaux/moderncv/issues/92

### CODE REVIEW

File: moderncv.cls [hunk 1]
```diff
@@ -266,18 +266,19 @@
...
+      \ifthenelse{\equal{#1}{linkedin}}     {\collectionadd[linkedin]{socials}     {\protect\httpslink[#3]{www.linkedin.com/in/#3}}}               {}%
+      \ifthenelse{\equal{#1}{xing}}         {\collectionadd[xing]{socials}         {\protect\httpslink[#3]{www.xing.com/profile/#3}}}              {}%
+      \ifthenelse{\equal{#1}{twitter}}      {\collectionadd[twitter]{socials}      {\protect\httpslink[#3]{www.twitter.com/#3}}}                   {}%
+      \ifthenelse{\equal{#1}{github}}       {\collectionadd[github]{socials}       {\protect\httpslink[#3]{www.github.com/#3}}}                    {}%
+      \ifthenelse{\equal{#1}{gitlab}}       {\collectionadd[gitlab]{socials}       {\protect\httpslink[#3]{www.gitlab.com/#3}}}                    {}%
+      \ifthenelse{\equal{#1}{stackoverflow}}{\collectionadd[stackoverflow]{socials}{\protect\httpslink[#3]{stackoverflow.com/users/#3}}}           {}%
+      \ifthenelse{\equal{#1}{bitbucket}}    {\collectionadd[bitbucket]{socials}    {\protect\httplink[#3]{www.bitbucket.org/#3}}}                  {}%
```
Comment (stephanlachnit) [inline moderncv.cls:275]: `\httpslink`

File: template.tex [hunk 2]
```diff
@@ -6,7 +6,7 @@
 % available at http://www.latex-project.org/lppl/.
 
 
-\documentclass[11pt,a4paper,sans]{./moderncv}        % possible options include font size ('10pt', '11pt' and '12pt'), paper size ('a4paper', 'letterpaper', 'a5paper', 'legalpaper', 'executivepaper' and 'landscape') and font family ('sans' and 'roman')
+\documentclass[11pt,a4paper,sans]{moderncv}        % possible options include font size ('10pt', '11pt' and '12pt'), paper size ('a4paper', 'letterpaper', 'a5paper', 'legalpaper', 'executivepaper' and 'landscape') and font family ('sans' and 'roman')
```
Comment (stephanlachnit) [inline template.tex:9]: not sure why there is a diff here

**Review by stephanlachnit — COMMENTED** [review]
Hm, somehow the checks are failing

File: template.tex (hunk 2 above)
Comment (ShadowMitia) [reply template.tex:9]: Right, I should've put that back... Sorry...

**Comment by ShadowMitia** [comment]
So it seems having `moderncv` and not `./moderncv` breaks the build. It's obvious when you think about it, but the errors were sooo cryptic...

**Comment by stephanlachnit** [comment]
Ahhh. Thanks for the PR :+1:

**Pull request merged**